aiven/resource_service.go

```diff
@@ -893,8 +954,14 @@ func resourceServiceState(ctx context.Context, d *schema.ResourceData, m interfa
 
 	err = copyServicePropertiesFromAPIResponseToTerraform(d, s, servicePlanParams, projectName)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("unable to copy api response into terraform schema: %w", err)
+	}
+
+	allocatedStaticIps, err := service.CurrentlyAllocatedStaticIps(ctx, projectName, serviceName, m)
+	if err != nil {
+		return nil, fmt.Errorf("unable to currently allocated static ips: %w", err)
 	}
+	d.Set("static_ips", allocatedStaticIps)
```

Comment:
and here, would be nice to have error handling

Reply:
:heavy_check_mark: